matrix/searching_in_sorted_matrix.py

```diff
@@ -1,26 +1,14 @@
-from typing import List
-
-
-def search_in_a_sorted_matrix(
-        mat: List[list], m: int, n: int, key: int) -> None:
-    '''
-    >>> search_in_a_sorted_matrix(\
-        [[2, 5, 7], [4, 8, 13], [9, 11, 15], [12, 17, 20]], 3, 3, 5)
-    Key 5 found at row- 1 column- 2
-    >>> search_in_a_sorted_matrix(\
-        [[2, 5, 7], [4, 8, 13], [9, 11, 15], [12, 17, 20]], 3, 3, 21)
-    Key 21 not found
-    '''
+def search_in_a_sorted_matrix(mat, m, n, key):
```

Comment:
@wuyudi  use static typing !!

Reply:
This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
So I undo it, and would make another pr after this.

Reply:
I've already done it in this. https://github.com/wuyudi/Python/pull/1/commits/18bedf489048146ef37039fcb6f5e473e8bd36b2